src/main/java/com/google/codeu/data/Marker.java

```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class Marker {
```

Comment:
Class description.

What does this describe/represent?
How and where do we use it? (seems like it is an immutable object)

Reply:
Description Added